### PR TITLE
Update general-requirements.adoc

### PR DESCRIPTION
**Pull request by mantel**
I'm not a native english speaker, but the wording does not "feel" correct to me...

### CODE REVIEW

File: modules/installation/pages/general-requirements.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // REMARK this list needs to be cleaned up
 Before you begin installation, ensure that your environment meets these requirements:
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion
Before you begin installation, ensure that you have:
```

File: modules/installation/pages/general-requirements.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
-* Current {scc} organization credentials
+* You have current {scc} organization credentials
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion
* Current {scc} organization credentials
```

File: modules/installation/pages/general-requirements.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
...
 * Access to installation media
 * Environment meets the hardware and networking requirements
 * You understand the supported client operating systems
```
Comment (Loquacity) [inline modules/installation/pages/general-requirements.adoc]:
```suggestion

You should also ensure that you understand the supported client operating systems.
```

**Comment by Loquacity** [comment]
You are absolutely correct, good spot! :+1: 

I have just gone through and reorganised the list to make it more readable.

File: modules/installation/pages/general-requirements.adoc (hunk 3 above)
Comment (keichwa) [reply modules/installation/pages/general-requirements.adoc]: I commited this suggestion separately

**Pull request merged**